leadiv/notification-banner-wc
# feature: basic notification

## What's New?
Ensures that the component will display a notification banner that has a user defined slot of banner. Ensures that an inactive property is set by the banner when the banner is not displayed.

## How To Test?
1. Run `npm test`
2. Ensure all tests pass.
3. Run `npm start`
4. Checkout the demo it should look like the screenshot below

## Screenshots
### Before
![screenshot from 2018-08-14 12-21-17](https://user-images.githubusercontent.com/965070/44104310-90e0b042-9fbc-11e8-8b98-19bb0bbba666.png)

### After
![screenshot from 2018-08-14 12-20-29](https://user-images.githubusercontent.com/965070/44104280-78af6248-9fbc-11e8-9693-9955735f9484.png)
